Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1160,6 +1160,15 @@ ExceptionOr<void> HTMLElement::showPopoverInternal(const HTMLElement* invoker)
     return { };
 }
 
+void HTMLElement::setInvoker(HTMLElement* popover, HTMLElement* invoker)
```

Comment:
You shouldn't need to pass the popover. `this` is the popover itself.

Reply:
in hidePopoverInternal we call this with nullptr, nullptr but I think you're right this is trying to handle a case that doesn't happen. Will double check quickly, but if it works I'll drop that param and push up soon.

Reply:
Fixed.